### PR TITLE
chore(package): update deps, fixes #2053

### LINKED ISSUE #2053: can't install respec
## Important info

* URL to affected spec: https://travis-ci.org/w3c/webdriver/builds/486466970?utm_source=github_status&utm_medium=notification
* ReSpec version: latest, see travis.yml
* [x] I did a "hard refresh", but it's still busted. 

## Description of problem

What happened (e.g., it crashed)?: npm failed to install highlight.js because of missing sha

But what I was expecting?: it to install...

Optional, steps to reproduce:
 1. npm i -g respec

the error log is 48 http fetch GET 200 https://registry.npmjs.org/snyk 1417ms
49 silly pacote range manifest for snyk@^1.122.0 fetched in 1429ms
50 silly resolveWithNewModule snyk@1.123.0 checking installable status
51 silly fetchPackageMetaData error for highlight.js@github:marcoscaceres/highlight.js#7b144b3b83ce296d5b29c5e4fc2ea2a65e9d3f32 Command failed: /usr/local/bin/git checkout 7b144b3b83ce296d5b29c5e4fc2ea2a65e9d3f32
51 silly fetchPackageMetaData fatal: reference is not a tree: 7b144b3b83ce296d5b29c5e4fc2ea2a65e9d3f32
52 timing stage:rollbackFailedOptional Completed in 0ms
53 timing stage:runTopLevelLifecycles Completed in 11921ms
54 verbose stack Error: Command failed: /usr/local/bin/git checkout 7b144b3b83ce296d5b29c5e4fc2ea2a65e9d3f32
54 verbose stack fatal: reference is not a tree: 7b144b3b83ce296d5b29c5e4fc2ea2a65e9d3f32
54 verbose stack
54 verbose stack     at ChildProcess.exithandler (child_process.js:297:12)
54 verbose stack     at ChildProcess.emit (events.js:197:13)
54 verbose stack     at maybeClose (internal/child_process.js:978:16)
54 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:265:5)
55 verbose cwd /Users/davidburns/development/mozilla-unified
56 verbose Darwin 18.2.0
57 verbose argv "/usr/local/Cellar/node/11.8.0/bin/node" "/usr/local/bin/npm" "install" "-g" "respec"
58 verbose node v11.8.0
59 verbose npm  v6.5.0
60 error code 128
61 error Command failed: /usr/local/bin/git checkout 7b144b3b83ce296d5b29c5e4fc2ea2a65e9d3f32
61 error fatal: reference is not a tree: 7b144b3b83ce296d5b29c5e4fc2ea2a65e9d3f32
62 verbose exit [ 1, true ]

### PR DESCRIPTION
**Pull request by marcoscaceres**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -88,13 +88,14 @@
     "command-line-usage": "^5.0.5",
     "epipebomb": "^1.0.0",
     "fs-extra": "^7.0.1",
-    "highlight.js": "github:marcoscaceres/highlight.js#7b144b3b83ce296d5b29c5e4fc2ea2a65e9d3f32",
+    "highlight.js": "git+ssh://git@github.com/highlightjs/highlight.js.git#cli",
```
Comment (saschanaz) [inline package.json]: Why SSH instead of previous `github:`?

Comment (marcoscaceres) [reply package.json]: No reason, just copy/pasted from GitHub... will switch, as the GitHub: urls are nicer to look at :)

File: package.json [hunk 2]
```diff
@@ -88,13 +88,14 @@
...
     "hyperhtml": "^2.25.4",
     "loading-indicator": "^2.0.0",
     "marked": "^0.6.0",
+    "npm-check-updates": "^2.15.0",
```
Comment (saschanaz) [inline package.json]: Is this intentional?

Comment (marcoscaceres) [reply package.json]: oh no, definitely not.

Comment (marcoscaceres) [reply package.json]: stupid `ncu` keeps asking to update, but locally... even though I have it installed globally, so keep screwing that up.

File: package.json [hunk 3]
```diff
@@ -88,13 +88,13 @@
     "command-line-usage": "^5.0.5",
     "epipebomb": "^1.0.0",
     "fs-extra": "^7.0.1",
-    "highlight.js": "github:marcoscaceres/highlight.js#7b144b3b83ce296d5b29c5e4fc2ea2a65e9d3f32",
+    "highlight.js": "github:highlightjs/highlight.js.git#cli",
```
Comment (marcoscaceres) [inline package.json]: I'm hoping this will go into hljs proper soon... waiting for code review.

Comment (marcoscaceres) [reply package.json]: https://github.com/highlightjs/highlight.js/pull/1951

**Pull request merged**